data/apps.volctl.gschema.xml

```diff
@@ -39,6 +39,11 @@
       <summary>OSD size</summary>
       <description>OSD scale in percent.</description>
     </key>
+    <key type="s" name="osd-position">
+      <default>"bottom-right"</default>
+      <summary>OSD position</summary>
+      <description>Select position for OSD volume; supported: center, top-right, bottom-right</description>
+    </key>
```

Comment:
Should use gsettings schema enum.

Options: IMO at least all 4 corners + center of the screen should be supported.
Even better 9 positions: Top, TR, Right, BR, Bottom, BL, Left, TL and Center